samples/esb/esb_prx/README.rst

```diff
@@ -0,0 +1,22 @@
+.. _esb_prx:
+
+Enhanced ShockBurst: Receiver
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample shows the Enhanced ShockBurst Receiver application.
+It includes the basic steps that are needed to transmit and receive packets using :ref:`ug_esb`.
+Follow the instructions and use the files from this page, and the :ref:`esb_ptx` sample.
+
+Requirements
+************
+
+.. note::
+   Follow the steps from and include the :ref:`esb_ptx` sample with this sample.
+
+.. include:: ../esb_ptx/README.rst
+   :start-after: esb_ptx_sample_requirements_start
+   :end-before: esb_ptx_sample_remaining_end
```

Comment:
I would argue against including `.. table-from-sample-yaml::` from another sample. One because I don't know if the script will read the contents from the correct sample. Two because this hides this line from the documentation.